tests/spec/s-function.js

```diff
@@ -1,7 +1,177 @@
 /* global describe, it, expect, beforeEach */
+var hasStrictMode = (function () {
+  'use strict';
+
+   return !this;
+}());
+
+var global = (function () {
+   return this;
+}());
 
 describe('Function', function () {
-    'use strict';
+    describe('#call()', function () {
+        var noop = function () {};
+        it('should not throw a error in each case', function () {
+            expect(function () {
+                noop.call();
+            }).not.toThrow();
+            expect(function () {
+                /*eslint no-useless-call: 0*/
+                noop.call(undefined);
+            }).not.toThrow();
+            expect(function () {
+                /*eslint no-useless-call: 0*/
```

Comment:
nbd, but i find it easier to grep for `/* eslint-disable no-useless-call */` and `/* eslint-enable no-useless-call */`